rwengine/src/render/OpenGLRenderer.cpp

```diff
@@ -17,6 +17,12 @@ constexpr GLuint kUBOIndexDraw = 2;
 
 GLuint compileShader(GLenum type, const char* source) {
     GLuint shader = glCreateShader(type);
+
+    if (shader == GL_ZERO) {
```

Comment:
Big nitpick but I think it should be `0` instead of `GL_ZERO` according to `glCreateShader`'s [documentation](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glCreateShader.xhtml). `GL_ZERO` is an enum value that just happens to be 0.